### PR TITLE
Adding some refinement of how to configure the environment

### PR DESCRIPTION
**Pull request by mgold1234**
## Please describe the change you are making

Adding some refinement of how to configure the environment for the first time.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

yes

## The code will be published under the BSD 3 clause license. Have you read and understood this license?

yes

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -57,6 +57,11 @@ ovirtclient.NewTestHelperFromEnv() function:
 
     helper := ovirtclient.NewTestHelperFromEnv(ovirtclientlog.NewNOOPLogger())
 
+you can also do it more simple and define the test helper with getHelper(t) function:
```
Comment (janosdebugs) [inline doc.go:60]: This documentation is meant for external users of this library. The `getHelper` function is not available when using go-ovirt-client as a library.

File: doc.go [hunk 2]
```diff
@@ -100,6 +105,8 @@ Storage domain to use for testing. Will be automatically chosen if not provided.
 
 VNIC profile to use for testing. Will be automatically chosen if not provided.
 
+if you want to connect to a live oVirt engine you need to define those environment variables in configuration,
```
Comment (janosdebugs) [inline doc.go:108]: This is also not relevant for external users, only internal ones.

**Review by janosdebugs — CHANGES_REQUESTED** [review]
The `doc.go` file is intended for external users of this library and is used to generate [this documentation](https://pkg.go.dev/github.com/ovirt/go-ovirt-client). The changes you are suggesting should go into the contributing guide, not the external documentation.

File: doc.go (hunk 2 above)
Comment (mgold1234) [reply doc.go:108]: ok, can I add those changes to the contributing guide?

Comment (janosdebugs) [reply doc.go:108]: Yes, if you find a place for it, feel free.

**Review by janosdebugs — APPROVED** [review]

**Comment by janosdebugs** [comment]
Thank you for the documentation update @mgold1234 we will merge this shortly, it seems we have a CI issue to sort out.

**Comment by janosdebugs** [comment]
@mgold1234 it seems your copy is not up to date with the upstream main branch. Could you please rebase onto the main?

**Comment by mgold1234** [comment]
@janosdebugs I didn't change any go file.
I did rebase to the main it it looks good when I run it locally.
can you try to rerun it again?

**Comment by janosdebugs** [comment]
@mgold1234 it seems you didn't push the rebase. Could you try and do that please?

**Comment by mgold1234** [comment]
@janosdebugs , I rebase it and run the golangci-lint locally.
do you have any idea why it's fail in the ci?

**Comment by mgold1234** [comment]
@janosdebugs, maybe I will create new pr? and delete this one to see if it solve the probelm?

**Pull request merged**